Documentation/ipvlan.md

```diff
@@ -38,3 +38,40 @@ Because all ipvlan interfaces share the MAC address with the host interface, DHC
 Therefore the container will not be able to reach the host via `ipvlan` interface.
 Be sure to also have container join a network that provides connectivity to the host (e.g. `ptp`).
 * A single master interface can not be enslaved by both `macvlan` and `ipvlan`.
+
+### Using ipvlan and dhcp
+There are two considerations when using "ipvlan" plugin together with "dhcp" IPAM plugin.
+
+First, since ipvlan driver uses the destination address in the IP packet when deciding to which slave the packet should be forwarded to, it will not work if the container interface is not yet assigned an IP.
+Since the DHCP packets are exchanged to determine which IP should be used, the interface will not yet have an IP address.
+A DHCP server typically unicasts the replies back to the client which will cause the ipvlan driver to drop the replies.
+To work around this, the DHCP client needs to instruct the DHCP server to use broadcasts instead.
```

Comment:
To work around this, the DHCP client must instruct the DHCP server to use broadcast replies instead.